redis/src/types.rs

```diff
@@ -147,44 +152,181 @@ pub enum Value {
     /// is why this library generally treats integers and strings
     /// the same for all numeric responses.
     Int(i64),
-    /// An arbitary binary data.
-    Data(Vec<u8>),
-    /// A bulk response of more data.  This is generally used by redis
+    /// An arbitrary binary data, usually represents a binary-safe string.
+    BulkString(Vec<u8>),
```

Comment:
I'm just looking at this in the context of the resp3 spec again. Should we call this `BlobString` to be more consistent w/ resp3 terminology?

Reply:
So, [here](https://github.com/redis/redis-specifications/blob/master/protocol/RESP3.md) it's called a blob string, while [here](https://redis.io/docs/reference/protocol-spec/) it's called a bulk string. I'm inclined to prefer the documentation over the .md file, but mostly because I'm lazy 🤷